app/js/components/policies/policiesModal/policiesModal.jade

```diff
@@ -0,0 +1,46 @@
+.policiesModal
+  .content-wrap
+    expand-all-button.toggle-expand-right
+    card(expandable='true', ng-repeat="policy in policies.resources | orderBy: 'policy.policy_id'", ng-show='!loading', init-collapsed='policy.policy_id != policyId')
+      card-header-expandable.gray
+        .card-header-title
+          .checks-wrapper
+            .title {{ policy.policy_id }}
+            .checks-state 
+              .passed(ng-if="policy.checks_pass !== 0") {{policy.checks_pass}} Checks passed
```

Comment:
This could use translation singular/plural support - i.e. 1 check vs 2 checks passed